### DISCUSSION TITLE
Development discussion category

### DISCUSSION THREAD

**Discussion by ollien** [vscode-neovim/vscode-neovim#1974] [Development]
Would anyone be opposed to me making a "Development" category, for ad-hoc discussions that need not specifically relate to an issue? It also might give a nice signal to new maintainers that they have a place they can ask questions about the codebase.

**Comment by justinmk** [comment]
No objection, but generally I would encourage development discussions to take the form of proposals , and having those in the issue tracker can be valuable.

**Reply by ollien** [reply]
For sure. My thinking is less for things like proposals and more questions about code or expected functionality. There's been a few times I've looked at something and had questions about intended behavior vs what the code does, or wanted to gather opinions (e.g. maybe I would have posted before making that lint rules PR recently)

**Comment by justinmk** [comment]
> questions about intended behavior vs what the code does

IMO those belong on the issue tracker

**Reply by ollien** [reply]
I don't see it as much different from Q&A about the tool, which we already push to discussions

**Reply by justinmk** [reply]
discussions are for:

- things that may never be resolved
- tech support
- opinions

 issue tracker is for project-related conclusions.
